diff -ur xf86-video-openchrome-857d892b668b4737d41ef1b7f58fd45eac84d552/src/via_display.c xf86-video-openchrome-857d892b668b4737d41ef1b7f58fd45eac84d552-xf86ModeStatusToString-hack/src/via_display.c
--- xf86-video-openchrome-857d892b668b4737d41ef1b7f58fd45eac84d552/src/via_display.c	2023-03-28 22:10:39.000000000 +0200
+++ xf86-video-openchrome-857d892b668b4737d41ef1b7f58fd45eac84d552-xf86ModeStatusToString-hack/src/via_display.c	2025-08-15 02:15:04.515516133 +0200
@@ -3195,6 +3195,10 @@
                         "Exiting %s.\n", __func__));
 }
 
+/* HACK: Copied prototype from xf86_priv.h */
+_X_EXPORT /* only for int10 module, not supposed to be used by OOT modules */
+const char * xf86ModeStatusToString(ModeStatus status);
+
 static Bool
 via_crtc_mode_fixup(xf86CrtcPtr crtc, DisplayModePtr mode,
                         DisplayModePtr adjusted_mode)
