diff -ur xf86-video-openchrome-857d892b668b4737d41ef1b7f58fd45eac84d552/src/via_driver.c xf86-video-openchrome-857d892b668b4737d41ef1b7f58fd45eac84d552-xf86newOption/src/via_driver.c
--- xf86-video-openchrome-857d892b668b4737d41ef1b7f58fd45eac84d552/src/via_driver.c	2023-03-28 22:10:39.000000000 +0200
+++ xf86-video-openchrome-857d892b668b4737d41ef1b7f58fd45eac84d552-xf86newOption/src/via_driver.c	2025-08-15 02:23:33.832375335 +0200
@@ -1125,7 +1125,7 @@
     }
 #endif /* OPENCHROMEDRI */
 
-    option = xf86NewOption(strEXAOptionName, strEXAValue);
+    option = xf86newOption(strEXAOptionName, strEXAValue);
     xf86CollectOptions(pScrn, option);
 
     viaSetupDefaultOptions(pScrn);
